src/tests/ftest/util/agent_utils.py

```diff
@@ -237,6 +284,30 @@ def dump_attachinfo(self):
                                    str(self.manager.job))[0]["stdout"]
         self.log.info("Agent attachinfo: %s", self.attachinfo)
 
+    def support_collect_log(self, **kwargs):
+        """Collect logs for debug purpose.
+
+        Args:
+            stop_on_error (bool, optional): Stop the collect-log command on very first error.
+            target (str, optional): Target Folder location to copy logs
+            archive (bool, optional): Archive the log/config files
+            extra_logs_dir (str, optional): Collect the Logs from given custom directory
+            target-host (str, optional): R sync all the logs to target system
+        Raises:
+            CommandFailure: if the daos_agent command fails.
+
+        Returns:
+            dict: the daos_agent command output converted to a python dictionary
```

Comment:
This returns `run_remote(...)`
```suggestion
        Returns:
            RemoteCommandResult: a grouping of the command results from the same hosts with the same return status
```